### PR TITLE
Fix scattering adjointness windows

### PR DESCRIPTION
**Pull request by tachella**
Fix failing CPU tests on windows due to adjointness test of scattering operator not passing on windows machines

### Checks to be done before submitting your PR

- [ ] `python3 -m pytest deepinv/tests` runs successfully.
- [ ] `black .` and `ruff check .` run successfully.
- [ ] `make html` runs successfully (in the `docs/` directory).
- [ ] Updated docstrings related to the changes (as applicable).
- [ ] Added an entry to the [changelog.rst](https://github.com/deepinv/deepinv/blob/main/docs/source/changelog.rst).

### CODE REVIEW

File: deepinv/loss/metric/perceptual.py [hunk 1]
```diff
@@ -21,13 +21,14 @@ class LPIPS(Metric):
...
     :Example:
 
-    >>> from deepinv.utils import load_example
-    >>> from deepinv.loss.metric import LPIPS
-    >>> m = LPIPS()
-    >>> x = torch.ones(2, 3, 32, 32)
-    >>> x_net = x - 0.01
-    >>> m(x_net, x) # doctest: +ELLIPSIS
-    tensor([...])
+    ::
```
Comment (Andrewwango) [inline deepinv/loss/metric/perceptual.py:24]: I fixed this doctest in #862 but we can merge whichever first

File: deepinv/models/third_party/promptir.py [hunk 2]
```diff
@@ -69,6 +69,19 @@ class PromptIR(Reconstructor, Denoiser):
...
     :param str pretrained: path to the pretrained weights or 'download' to download the authors' weights.
+
+    |sep|
+
+    :Example:
+
+    >>> import torch
+    >>> from deepinv.models import PromptIR
+    >>> model = PromptIR()
+    >>> x = torch.randn(1, 3, 256, 256)
```
Comment (Andrewwango) [inline deepinv/models/third_party/promptir.py:80]: is this not a bit heavy for doctest? can we make smaller?
